packages/phoenix-event-display/src/managers/three-manager/import-manager.ts

```diff
@@ -180,6 +182,13 @@ export class ImportManager {
     callback: (geometries: Object3D, eventData: Object3D) => void
   ): Promise<void> {
     const loader = new GLTFLoader();
+
+    const dracoLoader = new DRACOLoader();
+    dracoLoader.setDecoderPath(
+      `https://cdn.jsdelivr.net/npm/three@0.${REVISION}.0/examples/js/libs/draco/`
```

Comment:
I've just checked and your path is correct for 146... It seems things have changed on the threejs side !

Reply:
And here is the explanation : https://discourse.threejs.org/t/the-examples-js-directory-will-be-removed-with-r148/45349

Reply:
Right - updating `threejs` version is always a bit of work. Since phoenix is on [0.147.1](https://github.com/HSF/phoenix/blob/main/package.json#L28) I would definitely not recommend moving your local deployment to something ahead of that. If you need it for some reason, we can prioritise updating phoenix.

Reply:
My issue is that we did not intend to move our local deployment. I have to check why we do not have the default 147. @bencouturier any idea ?